app/views/results/_outgoings_table.html.slim

```diff
@@ -1,47 +1,48 @@
-= pdf_friendly_h2(t("results.show.client_outgoings"), "m", @is_pdf)
```

Comment:
I think you've accidentally deleted `= pdf_friendly_h2(t("results.show.client_outgoings"), "m", @is_pdf)` - this should be put back in above line 4.

Reply:
excellent spot @MazOneTwoOne , no idea how I lost that line and the way that github was displaying the changes made it difficult to see that had happened ( for me anyway)